sql/core/src/test/scala/org/apache/spark/sql/ColumnExpressionSuite.scala

```diff
@@ -1097,7 +1097,7 @@ class ColumnExpressionSuite extends QueryTest with SharedSparkSession {
             nullable = false))))
 
       structLevel2.withColumn("a", $"a".withField("a.b", lit(2)))
-    }.getMessage should include("Ambiguous reference to fields")
+    }.getMessage should include("Ambiguous reference to the field `a`")
```

Comment:
Please, use `checkError`, and don't check the error message. The idea of moving error formats to `error-classes.json` is to allow tech editors modify error messages w/o changing tests.

Reply:
Yep, done.